BlueprintUI/Sources/Layout/ElementBuilder.swift

```diff
@@ -7,6 +7,15 @@ extension ElementBuilder {
         [Child(element)]
     }
 
+    @available(
+        *,
+        unavailable,
+        message: "Optionals must be unwrapped with `if let value = ...` to be returned from a result builder."
```

Comment:
Is it necessary to have both of these? Shouldn't the first one suffice?

Reply:
One checks `Element` explicitly, and the other checks `Child`, which for most of these might be `OverlayChild`, etc. Without both, you could end up without the old error message.

Reply:
Oh right...forgot that Elements were implicitly converted.